packages/react-native-reanimated/src/css/platform/native/style/processors/__tests__/shadows.test.ts

```diff
@@ -2,8 +2,8 @@
 import type { BoxShadowValue } from 'react-native';
 
 import { processColor } from '../../../../../../Colors';
-import type { ProcessedBoxShadowValue } from '../shadows';
-import { processBoxShadow } from '../shadows';
+import { processBoxShadow } from '../../../../../../common/processors';
+import type { ProcessedBoxShadowValue } from '../../../../../../common/processors/shadows';
```

Comment:
```suggestion
import { processBoxShadow, type ProcessedBoxShadowValue } from '../../../../../../common';
```

Please add the `index.ts` file in the `src/common` directory to export everything that should be used outside the `src/common` directory instead of importing things directly from files nested within `src/common`.